src/util.rs

```diff
@@ -13,6 +12,7 @@ mod io_util;
 mod request_helpers;
 mod request_proxy;
 pub mod rfc3339;
+pub(crate) mod token;
```

Comment:
what's the reason for `pub(crate)`? wouldn't that make it hard to use from the integration test code?

Reply:
If tests actually end up needing access to `SecureToken` we could make the relevant things public. In general I prefer to keep as much as possible `pub(crate)` instead of `pub`, to enable unused warnings.

Reply:
> to enable unused warnings.

hmm, I could have sworn that I saw such warnings for `pub` code too when I was working on the session table stuff. but I'm okay with keeping it as `pub(crate)` for now until we actually need it.

Reply:
I see, thanks :)